src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -291,14 +302,35 @@ def _histogram_to_dynatrace_metric(self, metric: Metric,
             dict(point.attributes),
             int(point.time_unix_nano / 1000000))
 
+    def _sum_to_dynatrace_metric(self, metric: Metric, point: NumberDataPoint):
+        if metric.data.is_monotonic:
+            if metric.data.aggregation_temporality != \
+                    AggregationTemporality.DELTA:
+                self._log_temporality_mismatch(
+                    "monotonic Sum",
+                    metric,
+                    supported_temporality=AggregationTemporality.DELTA)
+                return None
+            return self._monotonic_to_dynatrace_metric(metric, point)
+        else:
+            if metric.data.aggregation_temporality != \
+                    AggregationTemporality.CUMULATIVE:
+                self._log_temporality_mismatch(
+                    "non-monotonic Sum",
+                    metric,
+                    supported_temporality=AggregationTemporality.CUMULATIVE)
+                return None
+            return self._non_monotonic_to_dynatrace_metric(metric, point)
+
     def _to_dynatrace_metric(self, metric: Metric, point: DataPointT):
         try:
             if isinstance(metric.data, Sum):
                 return self._sum_to_dynatrace_metric(metric, point)
             if isinstance(metric.data, Histogram):
                 return self._histogram_to_dynatrace_metric(metric, point)
             if isinstance(metric.data, Gauge):
-                return self._gauge_to_dynatrace_metric(metric, point)
+                # allow any temporality.
```

Comment:
```suggestion
                # gauge does not support or require temporality.
```

Reply:
changed in [dabe990](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/dabe9905bcfdc77cced8aba647032ef43af50f66). Thanks!